app_streaming.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import FastAPI
+from llama_cpp import Llama
+from transformers import pipeline
+from pydantic import BaseModel
+from fastapi.responses import StreamingResponse
+from typing import Optional
+
+# import uvicorn
+
+
+class Message(BaseModel):
+    text: str
+    temperature: Optional[float] = 0.3
+    max_tokens: Optional[float] = 100
+    top_p: Optional[float] = 0.1
+
+# echo = False
+# stop = ["Q", "\n"]
+
+
+instruct_model = Llama(model_path=r"<MODEL_PATH>")
+
+translation_model = pipeline("translation", model=r"<MODEL_PATH>", max_length=1000)
+app = FastAPI()
+
+
+@app.post("/summarize")
+async def summarize(message: Message):
+    translated_text = await translate(message.text, src_lang='heb_Hebr', tgt_lang='eng_Latn')
+    print("Finished translating")
+    message.text = f"Consider the following text: {translated_text}.\nA 5 points summary for the text is: "
+    return StreamingResponse(five_points_summary(message), media_type='text/event-stream')
+
+
+async def translate(text: str, src_lang: str, tgt_lang: str):
```

Comment:
The function doesn't do any async work (it doesn't have any `await` statement)
```suggestion
def translate(text: str, src_lang: str, tgt_lang: str):
```